test/libsolidity/smtCheckerTests/special/tx_data_immutable.sol

```diff
@@ -30,7 +36,10 @@ contract C {
 
 		fi();
 
-		assert(bhash == blockhash(12));
+		assert(blob_hash == blobhash(42));
```

Comment:
`blobhash` has a peculiarity: if `index >= len(tx.blob_versioned_hashes)`, then `blobhash(index)` should return 0, otherwise it should the versioned hash of transaction's index-th blob.

Reply:
Thanks for pointing this out.
I am not sure if we can model this in SMTChecker.

Reply:
After hints from @cameel, I have added a constraint to model the fact that number of blobs in a transaction is limited and reading beyond this limit will always return 0.
I have decided to use the pectra limit immediately, to avoid the need to come back to this and update it in a month or two.
This is OK since we overapproximate the real behaviour anyway.